scipy/signal/_peak_finding.py

```diff
@@ -257,24 +257,39 @@ def peak_prominences(vector, peaks, wlen=None):
         Indices of peaks in `vector`.
     wlen : int, optional
         A window lenght in samples that limits the search for the lowest
-        contour line two a symmetric interval around the evaluated peak. If
-        not given the entire vector is used. Use this parameter to speed up
-        the calculation significantly for large vectors.
+        contour line two a symmetric interval around the evaluated peak. If not
+        given the entire vector is used. Use this parameter to speed up the
+        calculation significantly for large vectors.
+    ret_bases : bool
+        If true, the peaks' left and right bases are returned. See returned
+        arguments below.
 
     Returns
     -------
     prominences : ndarray
         The calculated prominences for each peak in `peaks`.
+    left_bases, right_bases : ndarray, returned if ret_bases is true
+        The peaks base positions in `vector` to the left and right of each peak.
+        The higher base of each pair is a peak's lowest contour line (see Notes
+        for a more details).
 
     See Also
     --------
     find_peaks, peak_widths
 
     Notes
     -----
-    A contour line is defined as a horizontal line that intersects
-    the rising and falling peak slope in two points but contains no higher
-    second peak between these points.
+    Bellow the strategy to calculate a peak's prominence is summarized:
```

Comment:
Typo : Below

I think "strategy to compute a peak prominence is: "  is better.